contracts/characters.sol

```diff
@@ -252,6 +296,10 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
         return getPowerAtLevel(tokens[id].level);
     }
 
+    function getTotalPower(uint256 id) public view noFreshLookup(id) returns (uint256) {
```

Comment:
if we stored total power in an nft var on levelup and soul upgrade, we could save on calculating power for fights and other stuff that uses it (this will tie in to the equipment system too)
It's also important to discuss how pvp might use power, in case they don't want soul burns to affect their power, we might need separate values stored in which case the "totalpower" might require renaming to something like "pvePower"